R/model.R

```diff
@@ -929,6 +934,21 @@ CmdStanModel$set("public", name = "check_syntax", value = check_syntax)
 #'
 #' @examples
 #' \dontrun{
+#'
+#' # Example of fixing old syntax
+#' # real x[2] --> array[2] real x;
+#' file <- write_stan_file("
+#' parameters {
+#'   real x[2];
+#' }
+#' model {
+#'   x ~ std_normal();
+#' }
+#' ")
+#' mod <- cmdstan_model(file, compile = FALSE)
+#' mod$format(canonicalize = TRUE)
```

Comment:
I would strongly prefer the example for fixing old syntax to be
```
mod$format(canonicalize = list("deprecations"))
```
to keep the changes and possible git diffs minimal, when people want to just update the deprecated syntax.

I also suggest showing
```
mod$format(canonicalize = list("deprecations"), overwrite_file=TRUE)
mod$compile()
```

Reply:
Thanks, sounds good. I'll make the updates.